collector.go

```diff
@@ -277,6 +288,47 @@ func collectPerSubscriptionMetric(stats *stats, desc *prometheus.Desc, collectFu
 	})
 }
 
+func collectParkedMessagesPerSubscriptionMetric(stats *stats, desc *prometheus.Desc, ch chan<- prometheus.Metric) {
+	jp.ArrayEach(stats.subscriptionsStats, func(jsonValue []byte, dataType jp.ValueType, offset int, err error) {
+		eventStreamID, _ := jp.GetString(jsonValue, "eventStreamId")
+		groupName, _ := jp.GetString(jsonValue, "groupName")
+
+		// get parked messages max event number
+		parkedMessagesURL := fmt.Sprintf("/streams/$persistentsubscription-%s::%s-parked/head/backward/1", eventStreamID, groupName)
+		log.Info(parkedMessagesURL)
+		parkedMessagesResult := get(parkedMessagesURL, false)
+		eTagString, _ := jp.GetString((<-parkedMessagesResult).result, "eTag")
+		lastEventNumber, lastEventNumberErr := strconv.ParseInt(strings.Split(eTagString, ";")[0], 10, 64)
+		if lastEventNumberErr == nil {
+			lastEventNumber++ // +1 because Ids start from 0
+			log.Info(lastEventNumber)
+		}
+
+		// get $tb (truncate before) value from metadata
+		metadataURL := fmt.Sprintf("/streams/$persistentsubscription-%s::%s-parked/metadata", eventStreamID, groupName)
+		log.Info(metadataURL)
```

Comment:
Please review information you log in this method and the logging level. Probably most of it can be removed.